app/blocked/api/util.ts

```diff
@@ -0,0 +1,18 @@
+import axios from "axios";
```

Comment:
Move this file to `~/lib/ofac.ts`. We store utils general functions in the lib folder.

Reply:
https://github.com/Snowfork/snowbridge-app/pull/13/commits/5d1108714216e09694a5cfbeb3a8d8951452b7e9